### ISSUE TITLE
Does it support write to emmc now ?

### ISSUE THREAD

**Issue by liloew** [KryptonLee/e900v22c-CoreELEC#1]

**Comment by KryptonLee** [comment]
Use command `ceemmc -x` and choose single boot mode

**Comment by zzhydf** [comment]
Use command ceemmc -x and choose single boot mode

but:

Error opening file for writing '/dev/reserved'!
Failed to write the new partition table to eMMC!

**Comment by KryptonLee** [comment]
> Use command ceemmc -x and choose single boot mode
> 
> but:
> 
> Error opening file for writing '/dev/reserved'! Failed to write the new partition table to eMMC!

Sorry, I made a mistake about this as I have done it some time ago. I retried the installation process again by these steps just now (make sure your device has an Android OS on emmc before the installation process):
Step 1, use command `ceemmc -x` and choose dual boot mode.
Step 2, unplug the SD or USB storage device with CoreELEC OS and reboot the device to Android OS (not sure if this step is necessary or not).
Step 3, plug the SD or USB storage device with CoreELEC OS and reboot to CoreELEC OS.
Step 4, use command `ceemmc -x` and choose migrate the CoreELEC on emmc from dual boot mode to single boot mode.
